benchmark.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+type KV interface {
```

Comment:
I think this should be the tm-db interface. That we can test what is already in use and possibly how we can modify the interface to better suit the needs here.

Reply:
I created new interface for maximum flexibility (and no dependencies). For example I need snapshots, but don't need "Sync" version of functions.

Reply:
makes sense. I would hope the sdk would remove tm-db so this goes into my plans. 

![](https://media1.giphy.com/media/xl5QdxfNonh3q/giphy.gif)

Reply:
Let's firstly have a PoC and validate the approach with benchmarks.